README.rdoc

```diff
@@ -1,4 +1,4 @@
-= Warbler {<img src="https://badge.fury.io/rb/warbler.svg" alt="Gem Version" />}[http://badge.fury.io/rb/warbler] {<img src="https://travis-ci.org/jruby/warbler.png" />}[https://travis-ci.org/jruby/warbler]
+= Warbler {<img src="https://badge.fury.io/rb/warbler.svg" alt="Gem Version" />}[http://badge.fury.io/rb/warbler] {<img src="https://travis-ci.org/jruby/warbler.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/jruby/warbler]
```

Comment:
Display the current status of the `master` branch. Broken feature branches shouldn't reduce confidence in the main line.